lib/go-atscfg/atscfg.go

```diff
@@ -101,6 +106,15 @@ func ToServers(servers []tc.ServerV40) []Server {
 	return as
 }
 
+// ToServers converts a slice of the latest lib/go-tc and traffic_ops/vx-client type to the local alias.
+func ToServerUpdateStatuses(servers []tc.ServerUpdateStatusV40) []ServerUpdateStatus {
+	as := []ServerUpdateStatus{}
```

Comment:
Interesting. Reduces allocations since the `append` call later has to resize the slice as it grows? Should I do that for the other conversion functions declared before?

Fixed the names.